sdk/storage/Azure.Storage.Blobs/api/Azure.Storage.Blobs.netstandard2.0.cs

```diff
@@ -279,6 +288,7 @@ public partial class AppendBlobOpenWriteOptions
         public long? BufferSize { get { throw null; } set { } }
         public Azure.Storage.Blobs.Models.AppendBlobRequestConditions OpenConditions { get { throw null; } set { } }
         public System.IProgress<long> ProgressHandler { get { throw null; } set { } }
+        public Azure.Storage.UploadTransferValidationOptions TransferValidationOptions { get { throw null; } set { } }
```

Comment:
I also see `UploadTransferValidationOptions` as a property on `BlobClientOptions`.  Are the `*ValidationTransferOptions` client-level parameters, API-level parameters, or both?

Reply:
What happens if the API-level options and client-level options conflict?  Which takes precedence?

Reply:
method arg when not null takes precedence. An instance can be constructed with `None` as a disable.